test/python/test_examples.py

```diff
@@ -32,7 +32,6 @@
 class TestPythonExamples(QiskitTestCase):
     """Test example scripts"""
 
-    @unittest.skipIf(os.name == 'nt', 'Skip on windows until #2616 is fixed')
```

Comment:
Should this test still be skipped if only the limited charset is available? Or should we force Windows users to use a more general charset in that case? 🤔

Reply:
I hacked the way around https://github.com/Qiskit/qiskit-terra/pull/4900/commits/695bb4759dcf3180eb34def8def3b83754bc3c51